src/unix.rs

```diff
@@ -1,65 +1,40 @@
-extern crate libc;
-use std::os::raw::*;
-use std::os::unix::io::RawFd;
-
+#[cfg(unix)]
```

Comment:
I'm not sure this is necessary (since there is a cfg directive on the `mod unix` line in lib.rs).

But in either case, shouldn't this be `#![cfg(unix)]` and not `#[cfg(unix)]` ?

Reply:
Oh, yeah. I remember rustc was used to warn about this. But not anymore
so I often make this mistake.

Reply:
Oh, I see it now. I bind to the next `pub use` line.